conda_smithy/templates/README.md.tmpl

```diff
@@ -228,12 +228,31 @@ or with `mamba`:
 mamba install {{ ' '.join(outputs) }}
 ```
 
+or with [`pixi`](https://pixi.sh):
```

Comment:
I would like to omit having the global install as the most prominent pixi version. The vast majority of packages on conda-forge are modules where a global install doesn't make sense. 

Personally, I would not mention the global option at all.

Reply:
I agree with you
Thanks for the tip! I've removed it